modules/local/airrflow_report/airrflow_report.nf

```diff
@@ -6,8 +6,8 @@ process AIRRFLOW_REPORT {
         error "nf-core/airrflow currently does not support Conda. Please use a container profile instead."
     }
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
```

Comment:
I would simplify this to the suggestion below, as actually is the same image, but not a strong opinion (if agree also do it for the rest of modules).
```suggestion
    container "docker.io/immcantation/airrflow:4.1.0"
```